literature.bib

```diff
@@ -15240,13 +15317,26 @@ @inproceedings{YGM:SEAS06
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2005 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{EV:ICSM05,
+	author = {Andrew David Eisenberg and Kris De Volder},
+	title = {Dynamic Feature Traces: Finding Features in Unfamiliar Code},
```

Comment:
{{ }} For title. Why is the color encoding not working here? Any syntax errors? Cannot find one though.

Reply:
I think it is because we have comments in the file and Github does not parse these correctly. So Github thinks everything behind that comment is ill-formed. Fixed {{}}